release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -791,6 +826,8 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 [id="ocp-4-10-samples-bug-fixes"]
 ==== Samples
 
+* Before this update, if the Cluster Samples Operator encountered an `APIServerConflictError` error, it reported `sample-operator` as having `Degraded status` until it recovered. Momentary errors of this type aren't unusual during upgrades and were causing undue concern for administrators monitoring the Operator status. With this update, if the Operator encounters a momentary error, it no longer indicates `openshift-samples` as having `Degraded status` and retries to connect to the API server. Momentary shifts to `Degraded status` should no longer occur. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1993840[BZ#1993840])
```

Comment:
* Bug needs to be bold
* I think "status" shouldn't be in backticks, right? "`Degraded` status"? (3 places, if it is changed)
* aren't -> are not